test/webgl/framebuffer.spec.js

```diff
@@ -290,3 +290,33 @@ test('WebGL2#Framebuffer texture attach and read', t => {
   t.end();
 });
 */
+
+test('WebGL2#Framebuffer blit', t => {
```

Comment:
Awesome. 

Note for future PR: We should add a texture comparison helper method to validate the output.